Exiled.Example/Events/PlayerHandler.cs

```diff
@@ -128,6 +125,8 @@ public void OnDestroying(DestroyingEventArgs ev)
         /// <inheritdoc cref="Exiled.Events.Handlers.Player.OnDying(DyingEventArgs)"/>
         public void OnDying(DyingEventArgs ev)
         {
+            if (ev.Killer is null)
+                return;
             Log.Info($"{ev.Target.Nickname} ({ev.Target.Role}) is getting killed by {ev.Killer.Nickname} ({ev.Killer.Role})!");
```

Comment:
Log.Info($"{ev.Target.Nickname} ({ev.Target.Role}) is getting killed by {ev.Killer is null ? Himself : ({ev.Killer.Nickname} ({ev.Killer.Role})})!");